### DISCUSSION TITLE
Unable to export restricted instances on MAC OS 12.4 Monterey

### DISCUSSION THREAD

**Discussion by synthmania67** [RomanKubiak/ctrlr#478] [General]
Hi there, I'm struggling over this message exporting any instance as vst on MAC OS 12.4.
![Bildschirmfoto 2022-08-02 um 09 52 40](https://user-images.githubusercontent.com/110454569/182325697-bd5511ef-4c66-4d3f-9259-5a6ee903c49e.png)

What am I doing wrong?

**Comment by damiensellier** [comment]
You are doing nothing wrong, Monterey probably changed its way to connect apps to the finder Open/save window.
I saw the same error with another app using the native Save dialog.

Maybe you can try changing this preference in Ctrlr   MENU > EDIT > PREFERENCES : 

![untick the box](https://user-images.githubusercontent.com/55164385/182466208-cf547229-9b2c-486d-ba93-e604ed22552f.png)

untick the box "Use OS native file Open/save dialogs"

You'll probably get an ugly but working file browser export window instead.
Let us know if it's working this way. 

Are you Paolo from SynthMania?

**Reply by synthmania67** [reply]
Hi, thanks a lot. It works, but the vst3 has no menu.
No, I`m not Paolo. My website is "https://synthmania.de" :-)

**Reply by damiensellier** [reply]
Hi,
1. Can you please check if you unticked the "hide menu bar when exporting an instance" ? It's in the right side property pane.
![Capture d’écran, le 2022-08-03 à 13 53 56](https://user-images.githubusercontent.com/55164385/182676500-6aa8a129-02b1-4519-a849-23cc33843104.png)

If the setting must show the bar on the exported instance, you probably have the same problem as @puffer3 in this thread : https://github.com/RomanKubiak/ctrlr/discussions/459

2. exporting VST3 has some major naming problem. Unlike the VST2, VST3 instances' panel-names are not properly recognized in DAWs. VST3 exported instances cannot change their name and still have the stock "Instigator | Ctrlr" naming. So you have to export a VST3 blank instance from Xcode with the proper plugin name beforehand, and it's a pain in the *** to be honnest. This topic has been detailed here : 
https://github.com/RomanKubiak/ctrlr/discussions/394#discussioncomment-1875817

So, I highly recommand using VST2 as long as you can. Or you need to prepare a custom Ctrlr VST3 instance beforehand to go ahead in your project.

3. Paolo or not, you're very welcome on this board! have a good time with Ctrlr and don't hesitate to ask anything.

Damien

**Reply by synthmania67** [reply]
Hi Damien, sorry for my late reply. 
Now I tried different versions of CTRLR, but every version is buggy. Some versions don't export all graphics, another version is not compatible with newer systems or other things happen. I have no clue why there is no 100% working version available for MAC. 
I'll give up now...
At the original site "Ctrlr.org" I can't log in anymore. I saw there a discussion with an attached file which I need. But no log in, no file.

https://ctrlr.org/forums/topic/looking-for-the-basic-sysex-receive-send-panel/

All in all it is very frustrating.

**Reply by synthmania67** [reply]
![Bildschirmfoto 2022-08-11 um 15 19 28](https://user-images.githubusercontent.com/110454569/184142694-424fc093-0cf6-469c-b0cc-04e72c616a34.png)

**Comment by dnaldoog** [comment]
Hi @synthmania67 

this should be the file you are looking for!

https://sourceforge.net/projects/ctrlr-utilities/files/GSDR/

**Reply by synthmania67** [reply]
Thank you very much. Your tool helps me to understand some things.

**Comment by puffer3** [comment]
omg i had no idea there was a solution to this export issue. Thanks so much I dont have to bounce from my laptop anymore!! @damiensellier you are a boss
